### PR TITLE
BD-2290 SMS Double Opt-In

### PR DESCRIPTION
**Pull request by rachel-feinberg**
# Pull Request/Issue Resolution

Closes [#BD-2290](https://jira.braze.com/browse/BD-2290)

#### Is this change associated with a Braze feature/product release?
- [x] Yes (**EA TBD / GA TBD**)
- [ ] No

### CODE REVIEW

File: _docs/_hidden/private_betas/sms_double_opt_in.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+> The SMS double opt-in feature allows you to require users to explicitly confirm their opt-in intent before they can receive SMS messages. This helps you tailor your focus to users who are likely to be engaged or are engaged with SMS.
+
+When SMS double opt-in is turned on, users are sent an SMS message that asks for their explicit consent before they can be messageable by your campaigns or Canvases. 
+
+While not an explicit requirement of the Telephone Consumer Protection Act of 1991 (TCPA), Braze recommends that you configure SMS double opt-in to ensure users are aware and consenting to be a part of your SMS program. For more information about SMS Compliance, view [SMS laws, regulations, and abuse prevention]({{site.baseurl}}/user_guide/message_building_by_channel/sms/sms_laws_and_regulations/).
+
+{% alert important %}
+SMS double opt-in is currently in early access.<br>
+Contact your Braze customer success manager if you're interested in participating in the early access.
+{% endalert %}
```
Comment (lydia-xie) [inline _docs/_hidden/private_betas/sms_double_opt_in.md:22]:
```suggestion
{% alert important %}
SMS double opt-in is currently in early access. Contact your Braze customer success manager if you're interested in participating in the early access.
{% endalert %}
```

File: _docs/_hidden/private_betas/sms_double_opt_in.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+## SMS double opt-in workflows
+
+Users can provide their explicit consent through outbound or inbound SMS messages.
+
+### Outbound SMS Double Opt-In
```
Comment (lydia-xie) [inline _docs/_hidden/private_betas/sms_double_opt_in.md:28]:
```suggestion
### Outbound SMS double opt-in
```

File: _docs/_hidden/private_betas/sms_double_opt_in.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+When a user provides their phone number, they are sent an SMS message that asks for their consent.
+
+![Screenshot of inbound SMS message where a user sends "JOIN" and receives the response "Reply Y to confirm you want to JOIN our SMS program. 3msg/week, text STOP at any time to STOP, then texts back "Y".][1]{:style="max-width:40%;"}
+
+### Inbound SMS Double Opt-In
```
Comment (lydia-xie) [inline _docs/_hidden/private_betas/sms_double_opt_in.md:34]:
```suggestion
### Inbound SMS double opt-in
```

File: _docs/_hidden/private_betas/sms_double_opt_in.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+![Screenshot of outbound SMS message with the brand texting, "Welcome to BRAND text updates! 1 msg a week for the latest offers. Reply Y to opt-in.", the users replying with "Y", and the brand responding with "Thanks! You're now opted-in to BRAND alerts. Here is a promo code SMS10 for 10% off your first purchase!"][2]{:style="max-width:40%;"}
+
+## Subscription Group status
+
+Only after the user completes the SMS double opt-in workflow does their [Subscription Group status]({{site.baseurl}}/user_guide/message_building_by_channel/sms/sms_subscription_group/) update to `Subscribed`. If the user begins the workflow but doesn’t complete it, they remain `Unsubscribed` and cannot be sent SMS messages from that Subscription Group.
+
+Users can also be entered into the SMS double opt-in workflow if they are [subscribed from other sources]({{site.baseurl}}//user_guide/message_building_by_channel/sms/sms_subscription_group#how-users-sms-subscription-groups-get-set) (e.g., REST API, SDK).
+
+## Enabling SMS double opt-in
```
Comment (lydia-xie) [inline _docs/_hidden/private_betas/sms_double_opt_in.md:46]:
```suggestion
## Enabling SMS double opt-in
```
Small suggestion - maybe this should be the first H2 in the doc?

File: _docs/_hidden/private_betas/sms_double_opt_in.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+To turn on SMS double opt-in, navigate to the **SMS Global Keywords** table in the applicable Subscription Group, and click **Edit** in the **Opt-In Keyword Category**. Next, select your opt-in method (**Opt-In** or **Double Opt-In**). Selecting **Double Opt-In** will expand the page to show additional [configurable fields](#configurable-fields).
+
+![The Opt-In Method section has two opt-in methods to choose from: Opt-In and Double Opt-In.][3]{:style="max-width:50%;"}
+
+### Configurable fields {#configurable-fields}
+
+| Category   |    Fields    | Description   
+| ----------- |----------- |---------------- 
+| Opt-In Prompt | Keywords | These are the keywords that a user can text to indicate opt-in intent. `START` is a required keyword. This opt-in prompt will also be sent to the user when their subscription status is updated by sources listed in the [Subscription Sources](#subscription-sources) section.
```
Comment (lydia-xie) [inline _docs/_hidden/private_betas/sms_double_opt_in.md:56]:
```suggestion
| Opt-In Prompt | Keywords | These are the keywords that a user can text to indicate opt-in intent. `START` is a required keyword. This opt-in prompt will also be sent to the user when their subscription status is updated by sources listed in the [Subscription sources](#subscription-sources) section.
```

**Review by lydia-xie — CHANGES_REQUESTED** [review]
looks good - left some suggestions for your review!

File: _docs/_hidden/private_betas/sms_double_opt_in.md (hunk 4 above)
Comment (rachel-feinberg) [reply _docs/_hidden/private_betas/sms_double_opt_in.md:45]: Ooh yeah. That makes sense. I'll change that.

**Review by lydia-xie — APPROVED** [review]

**Review by bre-fitzgerald — APPROVED** [review]
lgtm!

**Pull request merged**